index.html

```diff
@@ -1644,6 +1644,147 @@ <h4>Background</h4>
+		message-based. Message-based systems decouple the relationship between producers of messages
+		and consumers of messages, creating an abstraction that puts the two on equal footing and allows
+		for many different kinds of devices with differing requirements to interface in a flexible
+		and technology-neutral way. DIDs allow people to simultaneously use the same identifier for
+		both traditional communications and newer transactions that require a higher level of cryptographic security,
+		as well as allowing them to switch contexts without losing any security guarantees. These decentralized identifiers are not
+		rented or borrowed from centralized providers but are instead directly under user control.
+        </p>
+      </section>
+      <section id="messagingDescription">
+        <h4>Description</h4>
+        <p>
+          Samira is a humanitarian aid worker in Khartoum who wants to create connections with new people she
+		meets. She has a smartphone but doesn’t have any internet access in remote areas that she visits
+		in her line of work. Since she can’t use the internet, most of her preferred methods of communication
+		are unavailable. When she meets a new person she wishes to communicate with, she establishes a
+		direct secure connection between her device and theirs by having them scan a QR code to pass on her
+		DID and start sharing messages secured by the keys referenced in her DID Document, which is served from
+		her device. She can maintain these secure connections whether she has access to the internet or not.
+		Whenever Samira is working on a new campaign or wants to raise awareness about issues affecting her
+		local region, she is able to securely communicate with all relevant parties, regardless of
+		the device, platform, or network infrastructure they are using.
+        </p>
+        <p>
+          At one point, Samira sees a friend of hers, Ahmed, who’s part of a local organization campaigning against unfair
+		labor practices in local agriculture. Samira wants to support the efforts of this group and get
+		involved, but she worries that publicly allying with the farm workers will negatively affect her
+		ability to act as a neutral party in labor negotiations. By creating a unique DID
+		for herself in the group at very little cost, she can engage in multi-party secure communication
+		with all of the members of Ahmed’s organization. Messages are secured via mutual authentication
+		between Samira and each of the other members. Using these connections, Samira is able to have private
+		and confidential communications with the other members at any time, without fear of censorship or
+		surveillance. Members of the organization are able to have confidence that they’re talking to
+		Samira, and she can verify who she is communicating with. Samira can use the same identifier and
+		connection with the group to share credentials that prove she’s a humanitarian worker and attest
+		to her history.
```

Comment:
whitespace and line lengths
```suggestion
At one point, Samira sees a friend of hers, Ahmed, who’s part of a
local organization campaigning against unfair labor practices in local
agriculture. Samira wants to support the efforts of this group and get
involved, but she worries that publicly allying with the farm workers
will negatively affect her ability to act as a neutral party in labor
negotiations. By creating a unique DID for herself in the group at very
little cost, she can engage in multi-party secure communication with
all of the members of Ahmed’s organization. Messages are secured via
mutual authentication between Samira and each of the other members.
Using these connections, Samira is able to have private and confidential
communications with the other members at any time, without fear of
censorship or surveillance. Members of the organization are able to have
confidence that they’re talking to Samira, and she can verify who she is
communicating with. Samira can use the same identifier and connection
with the group to share credentials that prove she’s a humanitarian
worker and attest to her history.
```